nym-vpn/ui/src/pages/index.ts

```diff
@@ -1,5 +1,8 @@
 export * from './home';
 export * from './settings';
+export * from './settings/display';
+export * from './settings/feedback';
+export * from './settings/legal';
 export { default as NavLayout } from './NavLayout';
```

Comment:
You should avoid exporting each sub-pages (cc https://github.com/nymtech/nym/pull/4279/files#r1441759234)